ingest/vivoapipw.py.example

```diff
@@ -0,0 +1,8 @@
+#Entries to access to vivo sparql update api
+# Copy this to vivoapipw.py and modify the entries
```

Comment:
Could add a comment here noting that this file is used by both shell and Python scripts